### PR TITLE
geoserver 2.7.1

### PR DESCRIPTION
**Pull request by waynegraham**
Updates geoserver to 2.7.1

### CODE REVIEW

File: Library/Formula/geoserver.rb [hunk 1]
```diff
@@ -2,8 +2,9 @@
 
 class Geoserver < Formula
   homepage 'http://geoserver.org/'
-  url 'https://downloads.sourceforge.net/project/geoserver/GeoServer/2.7.0/geoserver-2.7.0-bin.zip'
-  sha1 '04184225bd8e640642dbd275b1a87a14fdb89a8e'
+  version = '2.7.1'
```
Comment (DomT4) [inline Library/Formula/geoserver.rb]: Please revert this to how it was. We don't permit variables in the URL like this.

File: Library/Formula/geoserver.rb [hunk 2]
```diff
@@ -2,8 +2,9 @@
...
+  url "https://downloads.sourceforge.net/project/geoserver/GeoServer/#{version}/geoserver-#{version}-bin.zip"
+  sha1 '9843864a5fa3d0bf20a92bc548565e3cfab876c9'
```
Comment (DomT4) [inline Library/Formula/geoserver.rb]: If you could update this checksum to `sha256` when you revert the version change as well, that'd be :tada:. Thanks!

**Comment by waynegraham** [comment]
@DomT4 Is this better?

**Comment by mikemcquaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

**Pull request closed without merging**